releasenotes/notes/consistent-config-secret-decryption-f24dd89f5c485f88.yaml

```diff
@@ -0,0 +1,12 @@
+# Each section from every releasenote are combined when the
```

Comment:
There should probably be a separate release note for the bulk of the code in this PR. For this issue in this release note specifically - is the rebase to master OK? I see this release note in master already?

Reply:
This a part of an other cherry-pick of a merged PR. We'll rebase on master to reduce the noise

Reply:
Great, thank you, could you also please remove the `changelog/no-changelog` label from this PR? There should be a release note for these changes.

Reply:
Added changelog and removed label